4337/src/utils/safe.ts

```diff
@@ -98,6 +99,7 @@ export class MultiProvider4337 extends JsonRpcProvider {
     this.generalProvider = generalProvider
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Any is just a warning, right? Is it necessary to turn it off, or keeping it as a "todo" list is better?

Reply:
In this case, I don't think it is a `TODO` lint - as this is the function signature required by `ethers`, so it will always be the `any` and the lint is a false-positive.

That being said, I do agree with "keeping it as a "todo" list is better", so at least for the `eslint-disable-next-line @typescript-eslint/no-non-null-assertion` it would make sense to keep the lint warning around...